source/account_structure/index.html.md.erb

```diff
@@ -92,4 +92,8 @@ Within each gateway account for the PSP, you can edit:
 
 * which of your bank accounts your revenue goes to
 
-You should contact your PSP to find out more.
+You can either:
+
+* [contact us](/support_contact_and_more_information/#support) - if your PSP is Stripe
```

Comment:
Looks good to me - one thing is maybe it should be: * [contact us](/support_contact_and_more_information/#support) (if your PSP is Stripe) ... so having "if your PSP is Stripe" in brackets. A dash looks like there's more longer information to be read rather than just an aside bit of information, if that makes sense

Reply:
Thanks Pippa - I've amended now.